daml-assistant/integration-tests/src/DA/Daml/Assistant/IntegrationTests.hs

```diff
@@ -831,4 +831,37 @@ cantonTests = testGroup "daml canton-sandbox"
           , "--script-name Main:setup"
           , "--ledger-host=localhost", "--ledger-port=" <> show ledgerApiPort
           ]
+  , testCaseSteps "daml start --sandbox-canton" $ \step -> withTempDir $ \dir -> do
+      step "Creating project"
+      callCommandSilentIn dir $ unwords ["daml new", "skeleton", "--template=skeleton"]
+      step "Writing custom daml.yaml" -- need to drop the sandbox-options
+      writeFileUTF8 (dir </> "skeleton" </> "daml.yaml") $ unlines
+        [ "sdk-version: " <> sdkVersion
+        , "name: skeleton"
+        , "version: 0.0.1"
+        , "source: daml"
+        , "dependencies:"
+        , "  - daml-prim"
+        , "  - daml-stdlib"
+        , "  - daml-script"
+        , "init-script: Main:setup"
+        ]
+      step "Get a free port"
+      navigatorPort <- getFreePort
+      step "Run daml start"
+      withDamlServiceIn (dir </> "skeleton") "start"
+        [ "--sandbox-canton"
+        , "--sandbox-port=0"
+        , "--canton-admin-api-port=0"
+        , "--canton-domain-public-port=0"
+        , "--canton-domain-admin-port=0"
+        , "--open-browser=no"
+        , "--start-navigator=yes"
+        , "--navigator-port", show navigatorPort
+        ] $ do
+          waitForHttpServer
```

Comment:
should we check that the ledger is working, e.g., create a contract via the JSON API? Currently we’re just testing that navigator starts up which doesn’t say much.

Reply:
Agreed. I'm going to merge this for now and improve it after the break.